content/docs/5.1/deploy/operate/upgrade.md

```diff
@@ -38,18 +37,18 @@ Before upgrading your TiKV cluster, you first need to upgrade TiUP.
 Skip this step if one of the following situations applies:
 
 + You have not modified the configuration parameters of the original cluster. Or you have modified the configuration parameters using `tiup cluster` but no more modification is needed.
-+ After the upgrade, you want to use v5.0's default parameter values for the unmodified configuration items.
++ After the upgrade, you want to use default parameter values of v5.0 for the unmodified configuration items.
 {{< /info >}}
 
-1. Edit the topology file:
+1. Enter the `vi` editing mode to edit the topology file:
 
     ```bash
     tiup cluster edit-config <cluster-name>
     ```
 
 2. Refer to the format of [topology](https://github.com/pingcap/tiup/blob/release-1.4/embed/templates/examples/topology.example.yaml) configuration template and fill the parameters you want to modify in the `server_configs` section of the topology file.
 
-3. Save the changes and exit the editing mode.
+3. After the modification, <kbd>:</kbd> + <kbd>w</kbd> + <kbd>q</kbd> to save the change and exit the editing mode. Enter <kbd>Y</kbd> to confirm the change.
```

Comment:
I guess the readers (developers, system administrators, etc.) have already mastered the basics of `vi` editor.
```suggestion
After the modification, press <kbd>:</kbd> + <kbd>w</kbd> + <kbd>q</kbd> to save the change and exit the editing mode. Enter <kbd>Y</kbd> to confirm the change.
```

Reply:
You are right. I'll check with Mr. Gu Liangliang to decide whether to simplify the description.